content/chainguard/chainguard-images/videos/static-base-image.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Using the Chainguard Static Base Image"
+linktitle: "Static Base"
+lead: ""
+description: "Video demonstration of how to get the software version information from Chainguard Images"
+type: "article"
+date: 2023-08-29T15:21:01+00:00
+lastmod: 2023-08-29T15:21:01+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 10
+toc: true
+---
+
+{{< youtube ZT6177U0fUM >}}
+
+## Tools used in this video
+
+* [Docker](https://docker.com)
+* [Grype](https://github.com/anchore/grype)
+
+## Dockerfile
+
+```Dockerfile
+FROM cgr.dev/chainguard/go as build
+
+
+COPY main.go /main.go
+RUN CG_ENABLED=0 go build -o /hello /main.go
+
+
+FROM cgr.dev/chainguard/static
+COPY --from=build /hello /usr/local/bin/
+CMD ["hello"]
+```
+
+
+## Transcript
+
+So what's the best container base image to use?
+0:10
```

Comment:
```suggestion

0:10
```

The transcript section is being smushed all into a single line, so you can't really read it. My suggestions here are just to break it up so it's more legible

Reply:
Thanks for this! 

I never looked at the preview so I didn't notice - you should have just told me, it would probably have been quicker to fix on my end.

Reply:
This should be fixed in the version.